content/guides/hosting/hosting-options/dedicated_cloud/_index.md

```diff
@@ -52,8 +52,8 @@ Security controls for W&B Dedicated Cloud are periodically audited internally an
 
 ## Migration options
 
-Migration to Dedicated Cloud from a [Self-managed instance]({{< relref "/guides/hosting/hosting-options/self-managed/" >}}) or [Multi-tenant Cloud]({{< relref "../saas_cloud.md" >}}) is supported.
+Migration to Dedicated Cloud from a [Self-managed instance]({{< relref "/guides/hosting/hosting-options/self-managed/" >}}) or [Multi-tenant Cloud]({{< relref "../saas_cloud.md" >}}) is supported, subject to specific limits and migration-related constraints
```

Comment:
```suggestion
Migration to **Dedicated Cloud** from a [Self-managed instance]({{< relref "/guides/hosting/hosting-options/self-managed/" >}}) or [Multi-tenant Cloud]({{< relref "../saas_cloud.md" >}}) is supported, subject to specific limits and migration-related constraints.
```
OK< what are the constraints? If we can't list them, then maybe we should change this from the sentence you have to something like "Contact [support](mailto:support@wandb.com) for guidance on migrating to **Dedicated Cloud**.